kickstart/website/publish-rules.yaml

```
@@ -1,6 +1,6 @@
 summary: Publish the rules for the $SRYYYY game
 
-priority: major
+priority: should
```

Teams need to know what they're doing

```suggestion
priority: must
```